.github/workflows/release.yml

```diff
@@ -1,29 +0,0 @@
-name: release
```

Comment:
FYI this file is shown here because I had to push it to master before being able to test it in #2. GitHub will only show workflow_dispatch events if it is present in master. Anyway, just ignore this file for this PR and refer to #2 for more info on this.